packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -211,37 +207,11 @@ export const OutboundShipmentDetailViewComponent: FC = () => {
         </FormProvider>
       </Modal>
 
-      <AppBarContentPortal
-        sx={{
-          display: 'flex',
-          flex: 1,
-          justifyContent: 'center',
-          paddingLeft: '25px',
-        }}
-      >
-        <Box display="flex" flex={1}>
-          <InputWithLabelRow
-            label="label.customer-name"
-            Input={
-              <CustomerSearchInput
-                value={draft.name}
-                onChange={name => {
-                  draft.update?.('name', name);
-                }}
-              />
-            }
-          />
-        </Box>
-
-        <Box display="flex" flex={1} justifyContent="center">
-          <TabList value={currentTab} onChange={onChangeTab}>
-            <Tab value="general" label={t('label.general')} />
-            <Tab value="transport" label={t('label.transport')} />
-          </TabList>
-        </Box>
-
-        <Box display="flex" flex={1} justifyContent="flex-end" />
-      </AppBarContentPortal>
+      <OutboundShipmentToolbar
+        draft={draft}
+        onChangeTab={(val: string) => onChangeTab(val)}
```

Comment:
I don't see why the need to keep `onChangeTab` in the `DetailView` - it is only used in the toolbar. Are you planning for something later?

Reply:
The current tab is used for the tab context which is in the detail view. Could put the context into the toolbar and wrap all the content in it?

Reply:
ah, yeah. missed that. Could do, though it doesn't help too much. just wanting to slim down the `DetailView`